paprika/simulate.py

```diff
@@ -0,0 +1,160 @@
+        self.job['minimization']['GROUP'] =                     None    # or []
+
+        # Production input file settings
+        self.job['md'] = {}
+        self.job['md']['cntrl'] =                       OrderedDict()
+        self.job['md']['cntrl']['imin'] =               0
+        self.job['md']['cntrl']['ntx'] =                1
+        self.job['md']['cntrl']['irest'] =              0
+        self.job['md']['cntrl']['dt'] =                 0.002
+        self.job['md']['cntrl']['nstlim'] =             5000
+        self.job['md']['cntrl']['ntpr'] =               500
+        self.job['md']['cntrl']['ntwe'] =               0
+        self.job['md']['cntrl']['ntwr'] =               5000
+        self.job['md']['cntrl']['ntxo'] =               1
+        self.job['md']['cntrl']['ntwx'] =               500
+        self.job['md']['cntrl']['ioutfm'] =             1
+        self.job['md']['cntrl']['ntf'] =                2
+        self.job['md']['cntrl']['ntc'] =                2
+        self.job['md']['cntrl']['igb'] =                1
+        self.job['md']['cntrl']['cut'] =                999.0
+        self.job['md']['cntrl']['ntt'] =                3
+        self.job['md']['cntrl']['gamma_ln'] =           1.0
+        self.job['md']['cntrl']['ig'] =                 -1
+        self.job['md']['cntrl']['temp0'] =              298.15
+        self.job['md']['cntrl']['ntr'] =                None
+        self.job['md']['cntrl']['restraint_wt'] =       None
+        self.job['md']['cntrl']['restraintmask'] =      None
+        self.job['md']['cntrl']['nmropt'] =             1
+        self.job['md']['cntrl']['pencut'] =             -1
+        self.job['md']['ewald'] =                       None    # or OrderedDict() (Add the other namelists?)
+        self.job['md']['wt'] =                          None
+        self.job['md']['DISANG'] =                      'restraints.in'
+        self.job['md']['GROUP'] =                       None
+
+        #self.print_time = 1.0 #ps.  Figure out a way to update dicts if this is reset by user. 
+        # offer way for user to just provide the input file?
+
+    def minimize(self, Soft=False):
+        """
+        Minimize the system.
+
+            If Soft=True, slowly turn on non-bonded interactions during minimization
+            so that the restraints get enforced first.
+
+        """
+
+        if Soft:
+            self.job['minimization']['wt'] = [ "&wt type = 'NB', istep1=0, istep2=2000, value1 = 0.0, value2=0.0, IINC=50, /",
+                                                "&wt type = 'NB', istep1=2000, istep2=4000, value1 = 0.0, value2=1.0, IINC=50, /" ]
+
+        _write_input_file(self.path+'/gb_minimize.in', self.job['minimization'], title='GB Minimization.')
+
+        ### ToDo: Use nice error catching method dave figured out.
+        log.debug('Running GB Minimization at {}'.format(self.path))
+        sp.call("pmemd -O -p {0} -c {1} -ref {1} -i gb_minimize.in\
```

Comment:
Should output names be variables here too? Maybe make an note for future revisions so we don't leave in hardcoded things and forget.

Reply:
yeah, this is one of my checkpoints in the simulate issues thread.  seems like things could get real messy if we let the user give all their own files .... maybe not.  I'm definitely open to it.

Reply:
I'll think about it.

Reply:
Add to #18